ide/che-core-ide-ui/src/main/java/org/eclipse/che/ide/ui/smartTree/TreeStyles.java

```diff
@@ -36,6 +36,8 @@
 
         String infoTextContainer();
 
+        String vcsHeadLinkContainer();
```

Comment:
But this CSS is needed only for tree node container. It does not related to general html link.